promql/parse.go

```diff
@@ -281,12 +302,23 @@ func (p *parser) newBinaryExpression(lhs Node, op Item, modifiers Node, rhs Node
 	ret.RHS = rhs.(Expr)
 	ret.Op = op.Typ
 
+	// opRange returns the PositionRange of the operator part of the BinaryExpr.
+	// This is made a function instead of a variable, so it is lazily evaluated on demand.
+	opRange := func() (r PositionRange) {
+		//Remove whitespace at the beginning and end of the range
```

Comment:
Formatting.

Reply:
What exactly is wrong with formatting?

The unnecessary line breaks in the for loops are enforced by gofmt.

Reply:
No space after the //, no .

Reply:
Oh, I see.